string camera_id
---
int32 parts_amount
